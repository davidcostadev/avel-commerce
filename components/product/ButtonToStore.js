import React from 'react'
import PropTypes from 'prop-types'

const ButtonToStore = ({ to }) => (
  <a
    className="btn btn-lg btn-primary btn-buy"
    href={to}
  >
    Ir para loja
  </a>
)

ButtonToStore.propTypes = {
  to: PropTypes.string.isRequired,
}


export default ButtonToStore
