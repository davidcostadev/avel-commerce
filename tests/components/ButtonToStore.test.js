/* eslint no-undef: "off" */
import React from 'react'
import renderer from 'react-test-renderer'
import ButtonToStore from '../../components/product/ButtonToStore'

describe('<ButtonToStore />', () => {
  describe('render', () => {
    it('default', () => {
      const tree = renderer.create((
        <ButtonToStore
          to="http://domain/product"
        />
      )).toJSON()
      expect(tree).toMatchSnapshot()
    })
  })
})
