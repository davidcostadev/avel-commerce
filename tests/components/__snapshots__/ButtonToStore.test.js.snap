// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ButtonToStore /> render default 1`] = `
<a
  className="btn btn-lg btn-primary btn-buy"
  href="http://domain/product"
>
  Ir para loja
</a>
`;
